### PR TITLE
(feat) O3-4185: Add form for IPD admission

### PR DESCRIPTION
**Pull request by vasharma05**
Link to the ticket: https://openmrs.atlassian.net/browse/O3-4185

https://github.com/user-attachments/assets/185ce0a8-857f-4603-a7ab-ba1098c276f9

### CODE REVIEW

**Comment by vasharma05** [comment]
@mogoodrich @chibongho requesting your review.
Thanks!

File: distro/configuration/ampathforms/ipd_admission_request.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        }
+      ]
+    }
+  ],
+  "processor": "EncounterFormProcessor",
+  "encounter": "Transfer Request",
+  "encounterType": "Transfer Request",
+  "referencedForms": [],
+  "uuid": "xxxx",
+  "description": "Admission form IPD",
```
Comment (mogoodrich) [inline distro/configuration/ampathforms/ipd_admission_request.json:63]: Nit, but this an admission request, not an actual admisson (which doesn't happen until the until the request is selected and processed).

**Review by mogoodrich — APPROVED** [review]
LGTM, one small nitpick about what to put in the description, thanks @vasharma05

**Review by mogoodrich — APPROVED** [review]

**Review by gracepotma — COMMENTED** [review]
LGTM! <3 Awesome to have IPD admission functionality being added.

**Pull request merged**